.github/workflows/package-npm.yml

```diff
@@ -61,7 +65,7 @@ jobs:
     runs-on: ${{matrix.os}}
     strategy:
       matrix:
-        os: [ubuntu-latest, windows-latest, macos-latest]
+        os: [ubuntu-${{inputs.ubuntu-version}}, windows-latest, macos-latest]
```

Comment:
I'm getting
```
error parsing called workflow
".github/workflows/publish.yml"
-> "tree-sitter/workflows/.github/workflows/package-npm.yml@main" (source branch with sha:d82dd1521ffdb4b4b71377ffe4c4d7c2f3e15d18)
: You have an error in your yaml syntax on line 67
```

Reply:
I got reviewed
Reviewed so hard
Reviewed till it broke my code
🎵

Might need double quotes around the ubuntu one in the array...?

Reply:
Here's my PR for this - https://github.com/tree-sitter/workflows/pull/13